ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -920,3 +921,28 @@ def test_ipa_migrate_stage_mode_with_invalid_cert(self):
         )
         assert result.returncode == 1
         assert ERR_MSG in result.stderr_text
+
+    def test_ipa_hbac_rule_duplication(self):
+        """
+        This testcase checks that default hbac rules
+        are not duplicated on the local server when
+        ipa-migrate command is run.
+        """
+        run_migrate(
+            self.replicas[0],
+            "prod-mode",
+            self.master.hostname,
+            "cn=Directory Manager",
+            self.master.config.admin_password,
+            extra_args=['-n']
+        )
+        cmd = self.replicas[0].run_command(
```

Comment:
A nit I suppose but you aren't constructing a command to run here, you are actually running it. I'd switch 'cmd' with 'result' for clarity.